RELEASENOTES/v2.0.0.rc-2.MD

```diff
@@ -0,0 +1,68 @@
+# Notary v2 Release Candidate 2 
+
+## Overview
+Notary v2 working group is pleased to announce its second release candidate, RC-2. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v1.0.0-rc.2)](https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.2)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-rc.3)](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.3)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-rc.2)](https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.2)***   (Updated in this release) 
+- [Notary v2 Specs](https://github.com/notaryproject/notaryproject) ***[(notaryproject-v1.0.0-rc.2)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.2)*** (Updated in this release)
+
+## Goal of the release
+- Notary v2 specifications
+    - Updates to support signing and verification using the fallback method for referrers, as specified in the 1.1 version release candidate of the OCI Distribution specification. With this method, Notary v2 now supports a reference types solution but within the bounds of OCI 1.0 Image and Distribution specs 
+ - Update "Notation" (CLI client) implementation to match the latest specifications. With this new version of Notation client, users can
+    - Choose the method to manage signatures inside their registries - image manifest or artifact manifest.
+    - Provide optional meta-data at time of signing as key-value pairs.
+    - Use a new "inspect" command to see relevant details encoded in all signtures associated with an image.
+    - Use a new "completion" command to generate the corresponding completion script for their shells. 
```

Comment:
Is this new command? I think this is introduced by the CLI framework cobra @patrickzheng200  Could you confirm it?